app/models/mixins/scanning_mixin.rb

```diff
@@ -296,7 +296,7 @@ def scan_via_miq_vm(miqVm, ost)
         "status_code" => status_code.to_s,
         "message"     => scan_message
       )
-      save_metadata_op(xml_summary.to_xml.miqEncode, "b64,zlib,xml", ost.taskid)
+      save_metadata_op(MIQEncode.encode(xml_summary.to_xml.to_s), "b64,zlib,xml", ost.taskid)
```

Comment:
Can MIQEncode.encode just call .to_s itself, so it doesn't have to be the caller's responsibility?